lib/cloudregister/registerutils.py

```diff
@@ -717,6 +783,79 @@ def same_registry_auth_content(content, smt):
         return one_auths_key and auth_token in content.get('auths').values()
 
 
+# ----------------------------------------------------------------------------
+def unset_env_vars():
+    """Remove the registry environment variables."""
+    env_vars = ['REGISTRY_AUTH_FILE', 'DOCKER_CONFIG']
+    if not os.path.exists(BASHRC_LOCAL_PATH):
+        logging.info('%s file does not exist' % BASHRC_LOCAL_PATH)
+        # remove the enviroment variables from the env, if present
+        [os.environ.pop(env_var, {}) for env_var in env_vars]
+        logging.info(
+            '%s variables removed from the environment, if present'
+            % ' '.join(env_vars)
+        )
```

Comment:
Not necessary to access `os.environ`. How would the variables get set if the file from which they are set does not exists. Also it makes the assumption that it would matter in the parent process from where the clean up is executed. Something we cannot know and cannot predict. If the file is not there then there is nothing to do.

Reply:
> How would the variables get set if the file from which they are set does not exists

Could the user not set the environment variables, if they wanted to, manually ?